### ISSUE TITLE
Browser open sample welcome page

### ISSUE THREAD

**Issue by syjgino** [taunometsalu/ClustVis#12]
Hi,

I have run ClustVis container on windows successfully before. I recently updated docker and then removed the container and rerun it with the exact same command. 
`docker run -d --name customClustvisColor -p 7777:3838 -v C:\Users\baolongsu\Documents\customClustvis\ClustVis-master\:\srv\shiny-server\:ro taunometsalu/clustvis`
But this time it's not working. When I go to the address, it opens a sample shiny welcome page. Can you help me with it?

Many thanks!

**Comment by taunometsalu** [comment]
Hi, I would suggest to first check whether /srv/shiny-server/ has the right content:

`docker exec -ti customClustvisColor /bin/bash`
`cd /srv/shiny-server/`
`ls -l`

If ClustVis code is not there then there is something wrong with mounting the volume. Maybe something has changed in the new Docker version?

BTW, I have never seen backslash in the Linux folder context like in `\srv\shiny-server\`. Are you sure it was like this before, not `/srv/shiny-server/`?

**Comment by syjgino** [comment]
> Hi, I would suggest to first check whether /srv/shiny-server/ has the right content:
> 
> `docker exec -ti customClustvisColor /bin/bash`
> `cd /srv/shiny-server/`
> `ls -l`
> 
> If ClustVis code is not there then there is something wrong with mounting the volume. Maybe something has changed in the new Docker version?
> 
> BTW, I have never seen backslash in the Linux folder context like in `\srv\shiny-server\`. Are you sure it was like this before, not `/srv/shiny-server/`?

Hi taunometsalu,

Thank you for the quick response.

I tried to check the content, it returns:
`total 44`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 01_hello`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 02_text`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 03_reactivity`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 04_mpg`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 05_sliders`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 06_tabsets`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 07_widgets`
`drwxr-xr-x 3 root root 4096 Oct 10  2017 08_html`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 09_upload`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 10_download`
`drwxr-xr-x 2 root root 4096 Oct 10  2017 11_timer`
`lrwxrwxrwx 1 root root   38 Oct 10  2017 index.html -> /opt/shiny-server/samples/welcome.html`
`lrwxrwxrwx 1 root root   37 Oct 10  2017 sample-apps -> /opt/shiny-server/samples/sample-apps`

Then I tried to run container with "/", it worked out this time! And checking the content returns:
`total 0`
`drwxrwxrwx 1 root root 40960 Jun 26  2019 datasets`
`drwxrwxrwx 1 root root  4096 Jun 26  2019 docker`
`-rwxr-xr-x 1 root root 31970 Jun 27  2019 global-copy.R`
`-rwxr-xr-x 1 root root 32139 Apr  7 00:06 global.R`
`-rwxr-xr-x 1 root root  1453 Jun 26  2019 README.md`
`drwxrwxrwx 1 root root  4096 Jun 26  2019 Rpackage`
`-rwxr-xr-x 1 root root 45251 Jun 26  2019 server-copy.R`
`-rwxr-xr-x 1 root root 46432 Jun 27  2019 server.R`
`-rwxr-xr-x 1 root root 67908 Jun 26  2019 ui - Copy.R`
`-rwxr-xr-x 1 root root 68203 Jun 27  2019 ui.R`
`drwxrwxrwx 1 root root     0 Jun 26  2019 www`

I'm sure I used "\" before and it worked well. Maybe the new update no longer accept that?
